AWS/legos/aws_get_service_quota_detailsv1/aws_get_service_quota_detailsv1.py

```diff
@@ -0,0 +1,46 @@
+
+from __future__ import annotations
+
+from typing import Optional
+
+from pydantic import BaseModel, Field
+
+
+class InputSchema(BaseModel):
+    quota_code: Optional[str] = Field(
+        '"L-4FB7FF5D"',
```

Comment:
Should we have a default value? Since it is an optional field?

Reply:
its not optional